README.md

```diff
@@ -150,7 +150,7 @@ To stay up-to-date on our [public roadmap](https://github.com/orgs/i-am-bee/proj
 BeeAI framework is open-source and we ❤️ contributions.<br>
 
 To help build BeeAI, take a look at our:
-- [Python contribution guidelines](/python/docs/CONTRIBUTING.md)
+- [Python contribution guidelines](/python/CONTRIBUTING.md) 
 - [TypeScript contribution guidelines](/typescript/docs/CONTRIBUTING.md)
```

Comment:
thank you! and since you are in that code would you be able to also make similar update to Typescript link?